--- a/shim.c	2024-02-21 09:24:14.290465400 +0800
+++ b/shim.c	2024-02-21 09:43:20.239458100 +0800
@@ -897,7 +897,9 @@ static EFI_STATUS load_image (EFI_LOADED
 
 	efi_status = drive->OpenVolume(drive, &root);
 	if (EFI_ERROR(efi_status)) {
+		/*
 		perror(L"Failed to open fs: %r\n", efi_status);
+		*/
 		goto error;
 	}
 
@@ -906,7 +908,9 @@ static EFI_STATUS load_image (EFI_LOADED
 	 */
 	efi_status = root->Open(root, &grub, PathName, EFI_FILE_MODE_READ, 0);
 	if (EFI_ERROR(efi_status)) {
+		/*
 		perror(L"Failed to open %s - %r\n", PathName, efi_status);
+		*/
 		goto error;
 	}
 
@@ -1155,8 +1159,10 @@ EFI_STATUS read_image(EFI_HANDLE image_h
 		 */
 		efi_status = load_image(shim_li, data, datasize, *PathName);
 		if (EFI_ERROR(efi_status)) {
+			/*
 			perror(L"Failed to load image %s: %r\n",
 			       PathName, efi_status);
+			*/
 			PrintErrors();
 			ClearErrors();
 			return efi_status;
@@ -1209,7 +1215,9 @@ EFI_STATUS start_image(EFI_HANDLE image_
 	efi_status = handle_image(data, datasize, shim_li, &entry_point,
 				  &alloc_address, &alloc_pages);
 	if (EFI_ERROR(efi_status)) {
+		/*
 		perror(L"Failed to load image: %r\n", efi_status);
+		*/
 		PrintErrors();
 		ClearErrors();
 		goto restore;
@@ -1248,8 +1256,10 @@ EFI_STATUS init_grub(EFI_HANDLE image_ha
 	    efi_status == EFI_ACCESS_DENIED) {
 		efi_status = start_image(image_handle, MOK_MANAGER);
 		if (EFI_ERROR(efi_status)) {
+			/*
 			console_print(L"start_image() returned %r\n", efi_status);
 			usleep(2000000);
+			*/
 			return efi_status;
 		}
 
@@ -1261,18 +1271,22 @@ EFI_STATUS init_grub(EFI_HANDLE image_ha
 	// just fallback to the default loader.
 	if (!use_fb && (efi_status == EFI_INVALID_PARAMETER ||
 	                efi_status == EFI_NOT_FOUND)) {
+		/*
 		console_print(
 			L"start_image() returned %r, falling back to default loader\n",
 			efi_status);
 		usleep(2000000);
+		*/
 		load_options = NULL;
 		load_options_size = 0;
 		efi_status = start_image(image_handle, DEFAULT_LOADER);
 	}
 
 	if (EFI_ERROR(efi_status)) {
+		/*
 		console_print(L"start_image() returned %r\n", efi_status);
 		usleep(2000000);
+		*/
 	}
 
 	return efi_status;
@@ -1602,13 +1616,17 @@ load_unbundled_trust(EFI_HANDLE image_ha
 
 	efi_status = drive->OpenVolume(drive, &root);
 	if (EFI_ERROR(efi_status)) {
+		/*
 		perror(L"Failed to open fs: %r\n", efi_status);
+		*/
 		goto done;
 	}
 
 	efi_status = root->Open(root, &dir, PathName, EFI_FILE_MODE_READ, 0);
 	if (EFI_ERROR(efi_status)) {
+		/*
 		perror(L"Failed to open %s - %r\n", PathName, efi_status);
+		*/
 		goto done;
 	}
 
@@ -1655,8 +1673,10 @@ load_unbundled_trust(EFI_HANDLE image_ha
 				efi_status = root->Open(root, &dir, PathName,
 							EFI_FILE_MODE_READ, 0);
 				if (EFI_ERROR(efi_status)) {
+					/*
 					perror(L"Failed to open %s - %r\n",
 					       PathName, efi_status);
+					*/
 					goto done;
 				}
 				continue;
@@ -1681,8 +1701,10 @@ load_unbundled_trust(EFI_HANDLE image_ha
 			efi_status = root->Open(root, &dir, PathName,
 						EFI_FILE_MODE_READ, 0);
 			if (EFI_ERROR(efi_status)) {
+				/*
 				perror(L"Failed to open %s - %r\n",
 				       PathName, efi_status);
+				*/
 				goto done;
 			}
 		}
